storefront/widgets/objectedge/oePracticeDynamicAnant/templates/display.template

```diff
@@ -0,0 +1,21 @@
+<!-- oeES6Widget -->
+<div id="oeES6Widget">
```

Comment:
id name always be same with the widget name

Reply:
understood